components/director/internal/repo/exist.go

```diff
@@ -60,17 +62,20 @@ func (g *universalExistQuerier) unsafeExists(ctx context.Context, conditions Con
 		stmtBuilder.WriteString(" WHERE")
 	}
 
+	log.Debug("Add conditions to the query")
```

Comment:
We can add the actual conditions to the log so we can verify that the query that is executed below is right SQL representation of this conditions.